test/py/kubebench/test/deploy_utils.py

```diff
@@ -110,8 +110,7 @@ def setup_ks_app(test_dir, src_root_dir, namespace, github_token, api_client):
     util.run(["ks", "registry", "add", r, registries[r]], cwd=app_dir)
 
   # Install required packages
-  packages = ["kubeflow/core", "kubeflow/argo", "kubebench/kubebench-job",
-              "kubebench/nfs-server", "kubebench/nfs-volume", "kubeflow/tf-training"]
+  packages = ["kubeflow/argo", "kubeflow/kubebench", "kubeflow/tf-training"]
```

Comment:
You have to add kubeflow/common package as well. 
Argo uses this package.

Reply:
thanks! fixed it.